tailcfg/derpmap.go

```diff
@@ -87,6 +87,10 @@ type DERPRegion struct {
 	// "San Francisco", "Singapore", "Frankfurt", etc.
 	RegionName string
 
+	// RegionLatitude and RegionLongitude are optional float64 geographic coordinates of the region's city.
+	RegionLatitude  float64 `json:",omitempty"`
+	RegionLongitude float64 `json:",omitempty"`
```

Comment:
you could probably just say "Latitude" and "Longitude" here and below. (on the CityFoo variants)

The ones above have "Region" in front because those field names are used elsewhere and line up. But e.g. Avoid and Nodes below don't have "Region" prefixes.